scss/detail.scss

```diff
@@ -0,0 +1,36 @@
+.book-container {
+  font-family: 'Roboto', sans-serif;
+  padding: 15px;
+}
+
+.column {
+  display: flex;
+  flex-direction: column;
+}
+
+.row {
+  display: flex;
+}
+
+.book-title {
+  font-size: 32px;
+  font-weight: bold;
+}
+
+.book-gender {
+  color: $gray;
+  font-size: 24px;
+  font-weight: bold;
+  margin-left: 8px;
+  padding-bottom: 18px;
+}
+
+.book-property,
+.book-description {
+  padding-top: 18px;
+}
+
+.book-description {
+  margin-left: 8px;
+  padding: 18px 0;
```

Comment:
Same, use `margin-right` on sibling that is on the left side of this one. Delete padding.